ytemplates/bootstrap/value.spam_protection.tpl.php

```diff
@@ -15,6 +15,7 @@
             overflow: hidden;
             height: 1px;
             opacity: 1;
+	    padding-top: 1px;
```

Comment:
Ich denk man soll noch "box-sizing: border-box;" hinzufügen, um sicher zu stellen das nicht der Padding und die Height angezeigt werden. (Auch wenn dies standardmässig von Bootstrap aus geschieht.)

Reply:
Ich schlage vor, die Elemente so zu verstecken, wie Bootstrap das in der `visually-hidden`-Klasse macht: https://github.com/twbs/bootstrap/blob/180a0dc2e2336b78cbd6da98734c8639cdeaa6f7/scss/mixins/_visually-hidden.scss#L3-L18
Absolut positioniert nehmen sie keinen Raum ein, und mittels clip werden keinerlei Styles sichtbar.